ui/lib/kubernetes/addon/components/page/configuration.hbs

```diff
@@ -9,7 +9,7 @@
     <InfoTableRow @label="Kubernetes host" @value={{@config.kubernetesHost}} />
     {{#if @config.kubernetesCaCert}}
       <InfoTableRow @label="Certificate">
-        <CertificateCard @certificateValue={{@config.kubernetesCaCert}} />
+        <CertificateCard @certificateValue={{@config.kubernetesCaCert}} @isPem={{true}} />
```

Comment:
In Kubernetes configuration tab, when manual configuration is chosen, the user can enter a "Kubernetes CA Certificate" which should be a "PEM-encoded CA certificate to use by the secret engine...". Adding the argument @isPem={{true}} ensures that regardless of what the user enters, `<CertificateCard>` will say "PEM Format" for the label